project/project/plugins.sbt

```diff
@@ -0,0 +1,5 @@
+resolvers += Resolver.sonatypeRepo("releases")
+resolvers += Resolver.bintrayRepo("slamdata-inc", "maven-public")
+
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.1.0-M9")
+//addSbtPlugin("com.slamdata"    % "sbt-slamdata" % "2.4.1")
```

Comment:
leftover comment

Reply:
Also generally we prefer to use sbt-slamdata for the meta-builds. I guess I'm actually not a fan of that policy, but it's relatively standardized across our projects. What was the reasoning behind removing the plugin here?

Reply:
I believe it was oversight -- updated it in the latest commits I just pushed.

Reply:
The build broke when I uncommented it (please see https://travis-ci.com/slamdata/matryoshka/jobs/181335442#L2876).

Would appreciate any suggestions you might have on how to fix it. In the meantime, I pushed a new commit which reverts it (for comparison).